docs/cloud/managing-airbyte-cloud/manage-credits.md

```diff
@@ -72,4 +103,4 @@ To unenroll, [email us](mailto:billing@airbyte.io) with your request.
 Credits can no longer be pre-purchased. As of November 2024, Airbyte Cloud has moved to in-arrears billing invoiced monthly.
 :::
 
-Purchased credits expire after 1 year, or 12 months after purchase. Purchased credits are used before accuring an invoice.
\ No newline at end of file
+Purchased credits expire after 1 year, or 12 months after purchase. Purchased credits are used before accruing an invoice for additional usage. Purchased credits can not be used for the monthly subscription fee.
```

Comment:
Can we say "Purchased credits expire 12 months after purchase".

Reply:
Yes! Updated